### PR TITLE
Swagger docs added for Restock action.

### PR DESCRIPTION
**Pull request by nideyidayida**
image

### CODE REVIEW

File: app/server.py [hunk 1]
```diff
@@ -277,6 +277,26 @@ def restock_action(prod_id):
...
+    tags:
+        - Inventory
+    parameters:
+        - name: prod_id
+          in: path
+          description: ID of product.
+          type: integer
+          required: true
+        - name: restock_amt
+          in: path
```
Comment (MingshiPeng) [inline app/server.py]: I thought this parameter should be included in payload (i.e. in: data) instead of in path right?

Comment (jockyxu1122) [reply app/server.py]: image

Comment (jockyxu1122) [reply app/server.py]: You are right.
Changed from "path" to "data".

File: app/server.py [hunk 2]
```diff
@@ -277,6 +277,36 @@ def restock_action(prod_id):
...
+        -   Inventory
+    consumes:
+        -   application/json
+    definitions:
+        Restock_Amount:
+            type: object
+            properties:
+                restock_amt:
+                    type: integer
+                    minimum: 0
```
Comment (majia67) [inline app/server.py]: I guess for restocking, the minimum amount should be bigger than 0?

**Review by majia67 — COMMENTED** [review]
Why the restock_amt parameter in the screenshot is -1 by default? And why the example below shows the response fails?

File: app/server.py (hunk 2 above)
Comment (jockyxu1122) [reply app/server.py]: Our current restock function supports any restock_amt >= 0.

The restock_amt in the screenshot is manually typed in. There is no default value for that. The response should be 400 when restock_amt is -1. It's just a demonstration.

**Review by majia67 — APPROVED** [review]

**Comment by majia67** [comment]
IBM Cloud [toolchain](https://console.ng.bluemix.net/devops/toolchains/d4dd8efb-da17-4b0c-b4e6-5599ec73dd60?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18"): [Delivery Pipeline](https://console.ng.bluemix.net/devops/pipelines/c71be43a-8963-4052-b97c-5da9b99de3f9?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18") deployed [inventory](https://github.com/NYUDevOpsInventory/inventory/releases/tag/deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-dev-20180501-185525 "deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-dev-20180501-185525") to [dev](https://console.bluemix.net/apps/1db994d8-c4f1-4076-85c0-7ff2674244e5?env_id=ibm:yp:us-south "ibm:yp:us-south:NYU_Devops_Inventory_Org:dev"), including commit b004f601af37f51c319025273cad625ce636a979

**Comment by majia67** [comment]
IBM Cloud [toolchain](https://console.ng.bluemix.net/devops/toolchains/d4dd8efb-da17-4b0c-b4e6-5599ec73dd60?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18"): [Delivery Pipeline](https://console.ng.bluemix.net/devops/pipelines/c71be43a-8963-4052-b97c-5da9b99de3f9?env_id=ibm%3Ayp%3Aus-south "nyu-inventory-service-s18") deployed [inventory](https://github.com/NYUDevOpsInventory/inventory/releases/tag/deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-prod-20180501-185835 "deploy-ibm-yp-us-south-NYU_Devops_Inventory_Org-prod-20180501-185835") to [prod](https://console.bluemix.net/apps/e25e653b-a50f-4831-b93e-d8f9f484a484?env_id=ibm:yp:us-south "ibm:yp:us-south:NYU_Devops_Inventory_Org:prod"), including commit b004f601af37f51c319025273cad625ce636a979

**Pull request merged**